.github/workflows/create-design-review-issue.js

```diff
@@ -145,11 +151,81 @@ async function postGithub(token, url, data) {
   return JSON.parse(body);
 }
 
+async function graphqlQueryGithub(token, query) {
+  const {data} = await postGithub(token, '/graphql', {query});
+  return data;
+}
```

Comment:
Unfortunately, pinning and unpinning issues is only available via GraphQL, so we require some boilerplate to support them.